examples/sparseserver-ui/client/samples.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sample = {
+    "example 1": {
+        "context": """
+        The DeepSparse Engine is a CPU runtime that delivers 
+        GPU-class performance by taking advantage of sparsity within neural 
+        networks to reduce compute required as well as accelerate memory bound 
+        workloads. It is focused on model deployment and scaling machine 
+        learning pipelines, fitting seamlessly into your existing deployments 
+        as an inference backend.""",
```

Comment:
does this look fine in the running demo? AFAIK with the """ quote it includes any newlines or spaces used, so it may mess with the string's content. if so, I think this method might help
```lang=python
        "context": (
        "The DeepSparse Engine is a CPU runtime that delivers "
        "GPU-class performance by taking advantage of sparsity within neural "
        "networks to reduce compute required as well as accelerate memory bound "
        "workloads. It is focused on model deployment and scaling machine "
        "learning pipelines, fitting seamlessly into your existing deployments "
        "as an inference backend." ),
```